source/database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Alter Up/76_DML_PIMS_CLAIMS.sql

```diff
@@ -0,0 +1,30 @@
+BEGIN TRANSACTION;
+
+DECLARE @dbClaimsUserId nvarchar(30) = 'system';
+DECLARE @dbClaimsUserGuid uniqueidentifier = null;
+DECLARE @appClaimsUserId nvarchar(30) = N'SEED';
+DECLARE @appClaimsUserGuid uniqueidentifier = '1b93f614-91da-4b32-b36e-bd2c6ebd12e2';
+DECLARE @appClaimsUserDirectory nvarchar(30) = N'';
+DECLARE @seedClaimsTime DateTime = GETDATE();
+
+INSERT [dbo].[PIMS_CLAIM]
+    ([CLAIM_ID], [CLAIM_UID], [NAME], [KEYCLOAK_ROLE_ID], [DESCRIPTION], [IS_DISABLED], [CONCURRENCY_CONTROL_NUMBER], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [DB_CREATE_TIMESTAMP], [DB_CREATE_USERID], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY], [DB_LAST_UPDATE_TIMESTAMP], [DB_LAST_UPDATE_USERID])
+VALUES
+    (65, N'db03a38d-4eb4-41b1-b4f4-31e3355453f7', N'ltsa-ownership-view', N'db03a38d-4eb4-41b1-b4f4-31e3355453f7', N'Ability to view LTSA ownership information.', 0, 1, @seedClaimsTime, @appClaimsUserId, @appClaimsUserGuid, @appClaimsUserDirectory, @seedClaimsTime, @dbClaimsUserId, @seedClaimsTime, @appClaimsUserId, @appClaimsUserGuid, @appClaimsUserDirectory, @seedClaimsTime, @dbClaimsUserId);
```

Comment:
My comment for this (and the other SQL files) is that we should make sure that Doug does not override these in upcoming DB schemas. We've seen this happen in the past.